draft-ietf-rats-architecture.md

```diff
@@ -360,6 +363,42 @@ are signed.  For example, values might have been generated at boot, and then
 used in claims as long as the signer can guarantee that they cannot have changed
 since boot.
 
+The term freshness is applied to both Evidence and Attestation Results in
+this document. Unfortunately, the term freshness is also a bit overloaded and
+in order to illustrate the application of the term in this document better,
+two characteristics that can be part of freshness are elaborated on below:
+recentness and uniqueness:
+
+Recentness:
+
+: a quality of Evidence that is based on the time that has passed between
+creation of the evidence by an Attester (typically by "sealing" a Claims set
```

Comment:
In general this may not be correct since it may be about the time a claim value was generated rather than the time the evidence was generated.   Or perhaps maybe that's a third term and defining terms is not a good direction...

Reply:
pls, see the #88 

We have to "qualify" this step well. I am happy to read alternatives. It remains to be vital independent of phrasing, I think.

Reply:
I also think the time of reception is not particularly interesting, and the phrasing in 376 is not correct.  What's interesting is the time of appraisal, which is time(RG).